modules/learn/pages/clusters-and-availability/xdcr-conflict-logging-feature.adoc

```diff
@@ -0,0 +1,541 @@
+
+| `id`/`srcDoc.id`/`tgtDoc.id`
+| CRD document ID prefixed with `crd_`.
+
+A copy of conflict source document ID prefixed with `src_`.
+
+A copy of conflict target document ID prefixed with `tgt_`.
+
+| `docId`
+| document id of the actual document in conflict.
+
+| `replId`
+| Replication ID.
+
+| `srcDoc` and `tgtDoc`
+| Document ID of the “copy” of source document to look up in the conflict collection and the “copy” of the target document to look up in the conflict collection.
+
+| `nodeId`
+| Node identification, which can be the hostname or IP address and port.
+
+| `bucketUUID`
+| Source/target bucket unique ID.
+
+| `cas`
+| Source/target document CAS.
+
+| `clusterUUID`
+| Source/target cluster unique ID.
+
+| `isDeleted`
+| True or false for - is the source/target document deleted?
+
+| `scope` and `collection`
+| Name of the scope and collection where the conflict source/target documents exist.
+
+| `revSeqno`
+| Source/target document’s revision sequence number.
+
+| `expiry`
+| Source/target document’s expiry time.
+
+| `flags`
+| Source/target document’s flag.
+
+| `xattrs`
+| System xattributes.
+
+| `_vv`
+| Document version vector.
+HLV metadata.
+|===
+
+[#conflict-logger]
+=== Logger
+
+The Logger in XDCR records conflict events detected during a replication.
+It operates as an asynchronous component designed to minimize the impact on replication performance and system resources.
+
+Conflict logging is attempted on the best-effort basis.
+If any error occurs, due to resource issues or a delay in writing to the conflict log collection, XDCR may pause or stop the logging process, report the error type for the pipeline, and continue with replication so that there is no High Availability issue.
+
+By default, the Conflict Logger is tuned to handle a low percentage of conflicts.
+
+`xdcr_clog_status` is a Prometheus stat, which indicates the status of the Conflict Logger.
+The status can be one of the following:
+
+* Hibernated: The logger is temporarily disabled due to persistent errors or resource constraints.
++
+NOTE: Hibernation prevents the degradation of the replication performance. Logging is re-enabled after a set interval, or after the errors are manually resolved and the logger is manually restarted.
+
+* Running: The logger is actively processing conflict events.
+
+* DNE: The logger does not exist.
+
+[#configure-conflictlogging-settings]
+== Configuring the conflictLogging Setting
+
+You can configure a xref:learn:clusters-and-availability/xdcr-enable-crossclusterversioning.adoc#conflict-collection[conflict collection] by using the `conflictLogging` key. 
+
+The `conflictLogging` key is an advanced XDCR setting that enables or disables conflict logging and specifies where conflict records must be stored.
+It allows you to define a default conflict collection and, optionally, detailed logging rules (through `loggingRules`) for different scopes and collections.
```

Comment:
It allows you to define a default conflict log collection, and, optionally, granular logging rules for scopes and collections (via the `loggingRules` object).